backend/src/ml_space_lambda/utils/common_functions.py

```diff
@@ -150,7 +150,6 @@ def get_tags_for_resource(sagemaker, arn: str):
 def list_clusters_for_project(
```

Comment:
Name is probably a bit misleading at this point and it could be confusing given that  it's in common utils but in actuality has a very specific usecase.